docs/explanations/pro_token_and_machine_usage.rst

```diff
@@ -0,0 +1,49 @@
+.. _pro_token_and_machine_usage:
+
+Machine counts and token usage for Ubuntu Pro
+*********************************************
+
+Ubuntu Pro tokens are used to authenticate your machine and enable access to the services you are entitled to. These tokens are unique and specific to your account, so you should treat them as a secret (as you would with a password).
+
+In this page, we'll discuss how Ubuntu Pro tokens work and how machines are counted against your subscription.
+
+Ubuntu Pro token(s)
+--------------------
+
+- Obtaining Pro tokens:
+     After subscribing to Ubuntu Pro, you can access your subscriptions through the `Ubuntu Pro Dashboard <Pro_dashboard_>`_. Each subscription includes a Ubuntu Pro token that can be used to attach as many machines as the subscription allows.
+
+- Attaching Pro tokens:
+    Command: Use ``sudo pro attach`` to connect a machine to your Ubuntu Pro subscription.
+
+- Detaching Pro tokens:
+    Command: Use ``sudo pro detach`` to remove a machine from the subscription. If you are attached on a VM you intend to delete, the token does not need to be detached before the VM is destroyed.
+
+For further information regarding pro commands, refer to our :ref:`Getting started-with Ubuntu Pro client tutorial<tutorial-commands>`.
+
+Monitoring active machines:
+---------------------------
+
+Active machines are counted based on the number of machines attached to your subscription at a given time. An active machine is defined as a machine that has been attached to a given subscription at any point within the past 24 hours.
+
+Each subscription type has a limit on the number of machines it can cover. For personal subscriptions, this limit is 5 physical machines with unlimited VMs on these machines. Enterprise subscriptions will have a higher limit and may differentiate between physical and virtual machines.
+For use beyond 5 physical machines, consider an enterprise subscription.
+
+The Ubuntu Pro dashboard can help track the number of active machines, ensuring compliance with subscription limits. It provides a comprehensive view of all active and attached machines. It helps track the number of active machines, ensuring that you remain within the subscription limits.
+
+Dealing with multiple VMs:
+---------------------------
+
+In an IaaS environment where node control is limited, you can contact `Canonical's sales team <pro_sales_form_>`_ to discuss purchasing licenses per single VM rather than per physical node.
+
+
+Handling overage of active machines:
+------------------------------------
+
+Canonical monitors the number of active machines. Occasional overages are generally acceptable, but persistent overages may lead to subscription termination.
```

Comment:
I wonder if we need to expand this a little, to make it a bit less terse and discuss things in terms that I often see concerns/questions/anxieties expressed? There are many examples in the FAQ of people asking "what if I have X vms and I use the tokens..." so maybe it's worth using one of those examples as a hypothetical example to illustrate that we're not going to be knocking down their door if they accidentally have too many active machines? Something this vague is guaranteed to create more anxieties and send people to the FAQ to ask if their specific situation is going to get them in trouble/get their sub terminated.

Reply:
We can add a sub section about frequently asked questions about usage?
This can maybe hold these examples or atleast try to answer a more general concern about these active machines (based on the FAQs).

Reply:
My aim would be to try and reflect some of the language that is used in the questions about VMs and machine counts, rather than including an actual FAQ (since we have one already). By mirroring the same language and answering the questions in a more friendly way (the current sentence could be read as slightly unfriendly).

Reply:
Will do